active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,655 @@
+
+> Spec note: this is a new Xaml version of the existing 
+Windows.UI.Core.[VisibilityChangedEventArgs](https://docs.microsoft.com/uwp/api/Windows.UI.Core.VisibilityChangedEventArgs).
+
+
+### XamlVisibilityChangedEventArgs
+
+Contains the arguments returned by the event fired when a Window instance's visibility changes.
+
+> Spec note: this is a new XAML version of the existing 
+Windows.UI.Core.[VisibilityChangedEventArgs](https://docs.microsoft.com/uwp/api/Windows.UI.Core.VisibilityChangedEventArgs).
+
+**Handled**
+
+Gets or sets whether the window activation changed event was handled. **true** 
+if the event has been handled by the appropriate delegate; **false** if it has not.
+
+**Visible**
+
+Gets whether the window is visible or not. **true** if the window is visible; otherwise is **false**.
+
+## XamlWindowSizeChangedEventArgs class
+
+> Spec note: this is a new XAML version of the existing 
+Windows.UI.Core.[WindowSizeChangedEventArgs](https://docs.microsoft.com/uwp/api/Windows.UI.Core.WindowSizeChangedEventArgs).
+
+Contains the argument returned by a window size change event.
+
+### XamlWindowSizeChangedEventArgs Properties
+
+**Handled**
+
+Gets or sets whether the  event was handled. **true** if the event has been handled 
+by the appropriate delegate; **false** if it has not.
+
+**Size**
+
+Gets the new size of the window in units of effective (view) pixels.
+
+
+
+## XamlLaunchActivatedEventArgs class
+
+Provides event information when an app is launched.
+
+> Spec note: this is a new XAML version of the existing Windows.ApplicationModel.Activation.
+[LaunchActivatedEventArgs](https://docs.microsoft.com/uwp/api/Windows.ApplicationModel.Activation.LaunchActivatedEventArgs).
+
+## XamlLaunchActivatedEventArgs Properties
+
+**Arguments**
+
+Gets the arguments that are passed to the app during its launch. 
+In a UWP app, this is equivalent to LaunchActivatedEventArgs.Arguments.
+
+```CS
+public String Arguments { get; };
+```
+
+**LaunchActivatedEventArgs**
+Gets the 
+[LaunchActivatedEventArgs](https://docs.microsoft.com/uwp/api/Windows.ApplicationModel.Activation.LaunchActivatedEventArgs) 
+in a UWP app. This property is null in a Desktop app.
+
+```CS
+public Windows.ApplicationModel.Activation.LaunchActivatedEventArgs LaunchActivatedEventArgs { get; };
+```
+
+# API Details
+
+## Window class
+Microsoft.UI.Xaml namespace
+
+> The Window class' APIs in WinUI3 (**Microsoft**.UI.Xaml) match the existing system Window
+API (**Windows**.UI.Xaml), except for the following differences.
+
+```cs
+[webhosthidden]
+[contentproperty("Content")]
+unsealed runtimeclass Window
+{
+
+    ...
+
+    // These APIs match the system Xaml versions, except the event args are now from the Xaml namespace
+    event Windows.Foundation.TypedEventHandler<Object,Microsoft.UI.Xaml.XamlWindowActivatedEventArgs> Activated;
+    event Windows.Foundation.TypedEventHandler<Object,Microsoft.UI.Xaml.XamlWindowClosedEventArgs> Closed;
```

Comment:
Missing from spec. Did we intend to keep this? Or did we forget to delete it from the IDL?